src/lib/index.ts

```diff
@@ -355,6 +355,25 @@ class ServerlessIamPerFunctionPlugin {
         policyStatements.push(s);
       }
     }
+
+    // add iamPermissionsBoundary
+    if (!_.isEmpty(functionObject.iamPermissionsBoundary)
+      || !_.isEmpty(this.serverless.service.provider.iamGlobalPermissionsBoundary)){
```

Comment:
`iamGlobalPermissionsBoundary` property (unlike iamRoleStatements) is [not supported by serverless framework](https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/), so that I don't recommend to place it in provider level. Consider to move it under:
```
custom:
  serverless-iam-roles-per-function:
    iamGlobalPermissionsBoundary: arn:here..
```

Reply:
updated!! :)

Reply:
thanks, could you look at the rest of comments?